### ISSUE TITLE
Crash after requesting an autocrafting recipe -- Can't join my world anymore

### ISSUE THREAD

**Issue by ther4nd0moo** [refinedmods/refinedstorage#2627]
#### Steps to reproduce:
1. Launch Dungeons Dragons and Space Shuttles modpack through twitch and hit play
2. Wait for the title screen to load then try to hop on to my single player world
3. The game seems to load up, but shuts down immediately once its done and tells me it crashed

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.12.2
- Forge: 14.23.5.2847
- Refined Storage: 1.6.16

Does this issue occur on a server? [yes/no]
no
#### If a (crash)log is relevant for this issue, link it here:

[Crash Log](https://pastebin.com/tzvMj8WZ)

**Comment by Darkere** [comment]
The solution would be to open up the world/data/refinedstorage_nodes.dat in NBTExplorer and finding the Interface that is requesting this craft and removing its crafting upgrade.

**Comment by ther4nd0moo** [comment]
The main request I put in came from me requesting the item through the pattern grid. Is there something in the log that indicates this is coming from an interface? Because I can see how an interface could be requesting something needed for the main request.

**Comment by Darkere** [comment]
Yes the crashlog you posted says that it is an interface making the request.

**Comment by ther4nd0moo** [comment]
Oh wow! I didn't notice that.

I'm a bit new to NBTExplorer. I can find the interfaces and see that they have a refinedstorage:upgrade in their inventories. Would it be possible to know which one is requesting a craft? Or would it be better to just delete all the crafting upgrades?

**Comment by Darkere** [comment]
Unfortunately the crashlog doesn't say which interface is requesting what. If you want to be safe remove all of them.

**Comment by ther4nd0moo** [comment]
I'm back up and running again!! Thank you so much!

I located the interface and the recipe that was the culprit. 

I'm not sure if it helps with the devs here, but I believe this came from a recipe that I changed (using a different version of a slimeball. The interface that kept an item in stock needed this recipe, and I guess it didn't like that. I would assume that the interface would look for the recipe in all the crafters and use the current recipe, but ofc, I have no clue how it's implemented haha.

Thanks for helping me solve the issue and I love all the work that you all do!~
